Client/mods/deathmatch/logic/CClientVehicle.h

```diff
@@ -292,6 +292,9 @@ class CClientVehicle : public CClientStreamElement
     float GetHeliRotorSpeed();
     void  SetHeliRotorSpeed(float fSpeed);
 
+    float GetPlaneRotorSpeed();
```

Comment:
It is highly recommended to make Get methods **const**.

Reply:
all right, updated